week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,43 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
+// will return a number between 0 and 1 but not 1
```

Comment:
That's a good explanation of what `Math.random()` does - can you also explain what `getNumber()` does?